src/services/services.ts

```diff
@@ -3798,6 +3798,10 @@ module ts {
                 // Try getting just type at this position and show
                 switch (node.kind) {
                     case SyntaxKind.Identifier:
+                        if(isLabelName(node)){

```

Comment:
If there is no type, then "getTypeAtLocation" should not return anything.  if "getTypeAtLocation" returns something, we should show it.